Rakefile

```diff
@@ -1,6 +1,5 @@
-require "bundler/gem_tasks"
+# require "bundler/gem_tasks"
```

Comment:
I think this went away when I bumped the bundler version. I'm not sure I needed to do that, but I did so as a part of trying to figure out why I was having challenges with Docker. Regardless, the tests pass so I'm not worrying about it now.